File: Python/PopUpService.py
# popup_service.py

import tkinter as tk
from tkinter import messagebox

class PopUpService:
    @staticmethod
    def show_popup(title, message):
        root = tk.Tk()
        root.withdraw()  # Hide the main window
        if title == "Notification":
            messagebox.showinfo(title, message)
        else:
            full_message = message+"\n\nFor further assistance contact BioNX IT Team at BIONXMRLIT@msd.com"
            messagebox.showerror(title, full_message)
        root.destroy()  # Close the Tkinter window after showing the message

File: C#/PopupService.cs
using System.Windows.Forms;

public static class PopupService
{
    public static void ShowPopup(string message, string messageType)
    {
        MessageBoxIcon icon = messageType == "Notification" ? MessageBoxIcon.Information : MessageBoxIcon.Warning;
        if(messageType != "Notification" )
        {
             message += "\n\n For further assistance contact BioNX IT Team at BIONXMRLIT@msd.com";
        }
        MessageBox.Show(message, messageType, MessageBoxButtons.OK, icon);

        //Contact BioNX IT Team at BioNXMRLIT@msd.com //If error/warning
    }
}